external/storm-kafka-client/src/main/java/org/apache/storm/kafka/spout/KafkaTuple.java

```diff
@@ -15,22 +15,33 @@
  *   See the License for the specific language governing permissions and
  *   limitations under the License.
  */
-
 package org.apache.storm.kafka.spout;
 
-import org.apache.kafka.clients.consumer.ConsumerRecord;
-
-import java.util.List;
-
-public class KafkaSpoutTuplesBuilderWildcardTopics<K,V> implements KafkaSpoutTuplesBuilder<K,V> {
-    private KafkaSpoutTupleBuilder<K, V> tupleBuilder;
+import org.apache.storm.tuple.Values;
 
-    public KafkaSpoutTuplesBuilderWildcardTopics(KafkaSpoutTupleBuilder<K, V> tupleBuilder) {
-        this.tupleBuilder = tupleBuilder;
+/**
+ * A list of Values in a tuple that can be routed 
+ * to a given stream.
+ */
+public class KafkaTuple extends Values {
+    private static final long serialVersionUID = 4803794470450587992L;
+    private String stream = null;
+    
+    public KafkaTuple() {
+        super();
+    }
+    
+    public KafkaTuple(Object... vals) {
+        super(vals);
+    }
+    
+    public KafkaTuple routedTo(String stream) {
```

Comment:
This is because the constructor is varadic following the Values parent class. It is ambiguous to have

```
    public KafkaTuple(Object... vals)
```

and any other constructor.  Java complains.